scripts/build_wasm_samples.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e;
+SAMPLES_DIR="wasm_samples"
+TARGET_DIR="$SAMPLES_DIR/target/prebuilt"
+mkdir -p "$TARGET_DIR"
+# FIXME: `executor_custom_data_model` doesn't compile: https://github.com/hyperledger/iroha/issues/5114
```

Comment:
remove this as per [this](https://github.com/hyperledger/iroha/issues/5114#issuecomment-2392964395) comment

Reply:
Removed; building only crates with `cdylib` target.